src/components/Admin/AdminCardsSkeleton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Skeleton from 'react-loading-skeleton';
+
+const AdminCardsSkeleton = () => (
+  <div
+    className="admin-cards-skeleton mb-3 d-md-flex w-100"
+  >
+    <Skeleton />
+    <Skeleton />
+    <Skeleton />
+    <Skeleton />
```

Comment:
Might be good to rely on Bootstrap's existing grid here vs. customizing the width/margin of each skeleton with CSS, e.g.:

```jsx
<Row>
  <Col><Skeleton /></Col>
  <Col><Skeleton /></Col>
  <Col><Skeleton /></Col>
  <Col><Skeleton /></Col>
</Row>
```

Reply:
Oooooh, yes I like that plan.

Reply:
In the interest of time, I decided against this approach, but I would support us doing it in the future.